features/admin.ai.v1/context/ai-branding-feature-context.tsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { BrandingPreferenceInterface } from "features/admin.branding.v1/models";
```

Comment:
Let's use relative imports.

```suggestion
import { BrandingPreferenceInterface } from "../admin.branding.v1/models";
```

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5961/commits/891f646b2c63094d4467060253aa82c4bd6751b5